yarn.lock

```diff
@@ -7261,6 +7261,11 @@ nan@^2.12.1:
   resolved "https://registry.yarnpkg.com/nan/-/nan-2.14.1.tgz#d7be34dfa3105b91494c3147089315eff8874b01"
   integrity sha512-isWHgVjnFjh2x2yuJ/tj3JbwoHu3UC2dX5G/88Cm24yB6YopVgxvBObDY7n5xW6ExmFhJpSEQqFPvq9zaXc8Jw==
 
+nan@^2.14.0:
```

Comment:
Not sure why this got added. halp.

Reply:
I got that too, seems like someone didn't commit an updated yarn.lock file from some dependency update. Should be fine

Reply:
@blueridger This is from an reverted merge. Rebasing should eliminate it